Magento_Cms/styles/module/_home.scss

```diff
@@ -0,0 +1,41 @@
+.home {
```

Comment:
Technically, this should be a `home` view component, but for now, leave it as it is.